translations/es.ts

```diff
@@ -415,23 +415,23 @@
     <name>Downloads</name>
     <message>
         <source>Download Progress</source>
-        <translation>Progreso de descarga</translation>
+        <translation>Progreso de bajada</translation>
     </message>
     <message>
         <source>Download Map</source>
-        <translation>Descargar mapa</translation>
+        <translation>Mapas disponibles</translation>
     </message>
     <message>
         <source>Refresh</source>
```

Comment:
Just to be on the same page, this is button label visible when fetching of available maps from server fails... It is re-loading list of available maps, not updating downloaded maps...